packages/react/src/brands/hooks/useBrand.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  fetchBrand as fetchBrandAction,
+  getBrand,
+  getBrandError,
+  isBrandFetched,
+  isBrandLoading,
+  StoreState,
+} from '@farfetch/blackout-redux';
+import { useAction } from '../../helpers';
+import { useCallback, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import type { Brand } from '@farfetch/blackout-client';
+import type { UseBrandOptions } from './types/useBrand';
+
+const useBrand = (id: Brand['id'], options: UseBrandOptions) => {
```

Comment:
I think if it was named `brandId` would be easier to understand. What do you think?

Reply:
I agree too!